modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -34,7 +34,7 @@ def submit
 
         response = if intent_service.use_intent_api?
                      handle_210966_authenticated
-                   elsif form_is264555_and_should_use_lgy_api
+                   elsif params[:form_number] == '26-4555'
```

Comment:
Are there tests for this that you can add to or update?

Reply:
Great point! I've added a test. Thanks @rmtolmach